akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala

```diff
@@ -98,6 +98,15 @@ final class Source[+Out, +Mat](
     (mat, Source.fromPublisher(pub))
   }
 
+  /**
+   * Connect this `Source` to the `Sink.ignore` and run it. Elements from the stream will be consumed and discarded.
+   *
+   * Note that the `ActorSystem` can be used as the implicit `materializer` parameter to use the
+   * [[akka.stream.SystemMaterializer]] for running the stream.
+   */
+  def run()(implicit materializer: Materializer): Future[Done] =
```

Comment:
runWithIgnore/runIgnore?but run is ok

Reply:
I think `run` is too vague in what it means on a `Source` and that it should be clear from the name that it throws elements away.

Reply:
If we make the name longer than `run` I'm not sure it's worth it compared to `runWith(Sink.ignore)`.

The original motivation for #27789 is that it seems pretty common to have a side-effecting flow whose output is not always relevant, and in that case having to `.runWith(Sink.ignore)` reads pretty confusing: while it's of course technically correct that that's what the sink does, it doesn't really describe the intent of the user/code here.

Reply:
I think `run()` is nice. If you are interested in the elements the other `runX` are not far away.

Reply:
I still don't agree, but 2 vs 1 team votes is enough to move forward with it @jewertow

Reply:
Do you have a better suggestion? `runIgnore` is what we wanted to get away from. For new users it's confusing that you typically run a stream by "ignoring" it. I just want to run the thing. We often prefer Source and Flow, over Sink (e.g. in Alpakka).

Reply:
Nothing better than `runIgnore` or perhaps `runConsume`. IMO it's weird to talk about running on something that isn't terminated, like having run on a collection or `InputStream` so that's why I want something more indicating what it does with the elements.

I'm ok with being outvoted though.